modules/ddl-and-loading/pages/modifying-a-graph-schema.adoc

```diff
@@ -14,8 +14,8 @@ To safely update the graph schema, the user should follow this procedure:
 ====
 A schema change will invalidate any loading jobs or query which relate to an altered part of the schema. Specifically:
 
-* A loading job becomes invalid if it refers to a vertex or and an edge which has been *dropped* (deleted) or *altered* .
-* A query becomes invalid if it refers to a vertex, and edge, or an attribute which has been *dropped* .
+* A loading job becomes invalid if it refers to a vertex or and an edge which has been *dropped* (deleted) or *altered*.
+* A query becomes invalid if it refers to a vertex, and edge, or an attribute which has been *dropped*.
```

Comment:
We should always be careful to distinguish between individual vertices/edges vs. vertex/edge **types**.
Loading jobs refer to vertex/edge types; a job will be invalid if it refers to a type or an attribute of a type that no longer exists.

A query usually refers to **types** but in theory could refer to individual vertices and edges.  I think this rule is meant to also refer to types.

Reply:
2nd comment:
Use "DROP" and "DELETE" properly.
In SQL and GSQL, these two keywords have distinct meanings. Therefore, you should never say one is a synonym of the other.
DROP is to remove a catalog item. Catalog items include vertex TYPES, edge TYPES, graphs, queries, and loading jobs. [In a relational database, you could DROP a table.]

DELETE is to remove a data object from the database - a vertex or edge. [In a relational database you could DELETE a row of a table.]